daemon/images/image_builder.go

```diff
@@ -212,7 +212,7 @@ func (i *ImageService) GetImageAndReleasableLayer(ctx context.Context, refOrID s
 	}
 
 	if opts.PullOption != backend.PullOptionForcePull {
-		img, err := i.GetImage(refOrID, imagetypes.GetImageOpts{Platform: opts.Platform})
+		img, err := i.GetImage(ctx, refOrID, imagetypes.GetImageOpts{Platform: opts.Platform})
 		if err != nil && opts.PullOption == backend.PullOptionNoPull {
```

Comment:
That TODO comment below the fold should probably be addressed now that `err` could be `context.Canceled` or `context.DeadlineExceeded`.
https://github.com/moby/moby/blob/0db50996b76fa009b4e905dff35be3a1d308f698/daemon/images/image_builder.go#L219

Reply:
Done